dump.jsonld

```diff
@@ -0,0 +1,313 @@
+[
+  {
+    "@id": "https://ipaa.ca/profile/indigenous-organization/adaka",
+    "https://api.w.org/": [
+      {
+        "@id": "https://ipaa.ca/wp-json/"
+      }
+    ],
+    "@type": [
+      "http://schema.org/ItemPage"
+    ],
+    "http://schema.org/datePublished": [
+      {
+        "@type": "http://schema.org/Date",
+        "@value": "2019-08-06T14:25:20"
+      }
+    ],
+    "http://schema.org/lastReviewed": [
+      {
+        "@type": "http://schema.org/Date",
+        "@value": "2023-02-22T05:00:00"
+      }
+    ],
+    "http://schema.org/inLanguage": [
+      {
+        "@value": "en"
+      }
+    ],
+    "http://schema.org/primaryImageOfPage": [
+      {
+        "@value": "https://ipaa.ca/wp-content/uploads/ninja-forms/3/adaka.jpg"
+      }
+    ],
+    "http://schema.org/isPartOf": [
+      {
+        "@id": "https://ipaa.ca/indigenous-artists/"
+      }
+    ],
+    "http://www.w3.org/ns/prov#wasDerivedFrom": [
+      {
+        "@value": "https://ipaa.ca/profile/indigenous-organization/adaka"
+      }
+    ]
+  },
+  {
+    "@id": "https://ipaa.ca/indigenous-artists/",
+    "@type": [
+      "http://schema.org/CreativeWork"
+    ],
+    "http://schema.org/name": [
+      {
+        "@value": "Indigenous Performing Arts Alliance Membership Directory"
+      }
+    ],
+    "http://schema.org/sameAs": [
+      {
+        "@id": "http://www.wikidata.org/entity/Q113164737"
+      }
+    ],
+    "http://www.w3.org/ns/prov#wasDerivedFrom": [
```

Comment:
The derivedFrom for CreativeWork has multiple values.

Reply:
Regarding this, please make the following changes:
1. Only add the triple <subject> prov:wasDerivedFrom <page_url> to **top level blank nodes** on the web page.  This is because we are creating a new entity (new URI) for these. It the entity already as a URI (@id) then we don't need to add the prov:wasDerivedFrom triple.
2. The object <page_url> must be a URI instead of a Literal.